media/css/mozorg/locales.scss

```diff
@@ -0,0 +1,91 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import '../../protocol/css/includes/lib';
+
+.c-simple-header {
+    @include at2x('/media/img/mozorg/locales/world-map.png', 179px, 90px);
+    background-position: top center;
+    background-repeat: no-repeat;
+    padding-top: 90px + $spacing-md;
+    margin-top: $layout-xs;
+    text-align: center;
+
+    .c-simple-header-title {
+        @include text-display-md;
+    }
+
+    @media #{$mq-md} {
+        margin-top: $layout-lg;
+    }
+}
+
+.c-block-list {
+    margin: $layout-md 0;
+
+    .c-block-list-title {
+        @include text-display-sm;
+        border-bottom: 1px solid $color-gray-30;
+        padding-bottom: $spacing-md;
+    }
+
+    a:link,
+    a:visited {
+        color: $color-black;
+        text-decoration: none;
+    }
+
+    a:hover,
+    a:visited,
+    a:focus {
+        text-decoration: underline;
+    }
+
+    ul {
+        margin: $spacing-xl 0;
+
+        li {
+            margin-bottom: $spacing-lg;
+        }
+    }
+
+    @media #{$mq-sm} {
+        ul {
+            @include grid-gap($spacing-lg $spacing-xl);
+            display: grid;
```

Comment:
Note for reviewer: as this page is so simple, I decided not to include a float/flex based fallback for browsers that don't support grid. Older browsers will just get a simple vertical list. I figured that this page is mainly for robots, so shouldn't matter too much. Happy to add a fallback if folks disagree, or we could update this page later to use the upcoming Protocol grid mixins.

Reply:
👍